src/test/run-pass/rvalue-static-promotion.rs

```diff
@@ -0,0 +1,17 @@
+// Copyright 2017 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+#![feature(rvalue_static_promotion)]
```

Comment:
Is this even run when I run `./x.py test src/test/run-pass --test-args rval`?
I see `test [run-pass] run-pass/rvalue-static-promotion.rs ... ok` in the output but it would seem magical if it did. I guess it is set up to be magical because other tests do it the same way, but wonder where that magic is documented.